Dev-Corner/tutorials/files/hands_on-visualizations/alignment_rname_boxplot/templates/alignment_rname_boxplot.mako

```diff
@@ -14,15 +16,28 @@
     ## Actual file URL:
     file_url = os.path.join(root, 'datasets', hdadict['id'], "display?to_ext="+hda.ext)
 
-    ## Ensure BAI index is symlinked
-    bai_target = hda.file_name+'.bai'
+    # File path in tmp dir:
+    temp_data_dir = tempfile.gettempdir()+'/plugins/visualizations/'+visualization_name
```

Comment:
@yhoogstrate is there any need to make the path so complicated?
It could just be `tempfile.gettempdir()/foo.bam` isn't it. Even hardcoded.
If you don't agree maybe you can use os.path.join() ?

Thanks for fixing this!